up_fast_downward/fast_downward_grounder.py

```diff
@@ -64,6 +64,8 @@ def supported_kind() -> ProblemKind:
         supported_kind.set_effects_kind("FLUENTS_IN_BOOLEAN_ASSIGNMENTS")
         supported_kind.set_quality_metrics("ACTIONS_COST")
         supported_kind.set_actions_cost_kind("STATIC_FLUENTS_IN_ACTIONS_COST")
+        supported_kind.set_actions_cost_kind("INT_NUMBERS_IN_ACTIONS_COST")
+        supported_kind.set_actions_cost_kind("REAL_NUMBERS_IN_ACTIONS_COST")
```

Comment:
Thanks for the feedback, I am updating. Does the `FastDownwardGrounder` support `REAL_NUMBERS_IN_ACTIONS_COST` or is it also to remove? (around line 180)

The problem with the tests is not the `misc/tests/test-simple.py` file, but the fact that the tests are using the last stable version of the UP (where `INT_NUMBERS_IN_ACTIONS_COST` is not defined). The tests would pass if it can use the version that is currently in PR (https://github.com/aiplan4eu/unified-planning/pull/449) but I don't see any clear way of doing that without the commit id (or name of the branch) becoming obsolete soon.